node/src/mpc_client.rs

```diff
@@ -156,36 +156,42 @@ impl MpcClient {
                                 }
                                 MpcTaskId::Signature {
                                     id,
-                                    presignature_id,
+                                    presignature_id
                                 } => {
                                     metrics::MPC_NUM_PASSIVE_SIGN_REQUESTS_RECEIVED.inc();
                                     // TODO(#69): decide a better timeout for this
                                     let SignatureRequest {
                                         msg_hash,
                                         tweak,
                                         entropy,
+                                        key_type,
                                         ..
                                     } = timeout(
                                         Duration::from_secs(config.signature.timeout_sec),
                                         sign_request_store.get(id),
                                     )
                                     .await??;
                                     metrics::MPC_NUM_PASSIVE_SIGN_REQUESTS_LOOKUP_SUCCEEDED.inc();
-
+                                    
                                     timeout(
                                         Duration::from_secs(config.signature.timeout_sec),
-                                        sign(
-                                            channel,
-                                            client.my_participant_id(),
-                                            root_keyshare.keygen_output(),
-                                            presignature_store
-                                                .take_unowned(presignature_id)
-                                                .await?
-                                                .presignature,
-                                            msg_hash,
-                                            tweak,
-                                            entropy,
-                                        ),
+                                        match key_type {
+                                            KeyType::ED25519 => todo!(),
```

Comment:
same here and below, please specify an issue number.
I would probably prefer
```
{
  // TODO(#119): implement ED25519 signing
  unreachable!()
}
```
clearly indicating that this feature is unimplemented but is currently not triggerable in the code.